client/test/cypress/fixtures/executions_history.json

```diff
@@ -0,0 +1,327 @@
+  "buildDateTime": "2020-02-27T07:55:00.000Z",
+  "testDateTime": "2020-02-27T08:00:00.000Z",
+  "jobUrl": "https://build.company.com/demo/develop/day/4/",
+  "status": "DONE",
+  "result": "SUCCESS",
+  "acceptance": "NEW",
+  "blockingValidation": true,
+  "qualityThresholds": {
+    "sanity-check": {
+      "failure": 100,
+      "warning": 100
+    },
+    "high": {
+      "failure": 95,
+      "warning": 98
+    },
+    "medium": {
+      "failure": 90,
+      "warning": 95
+    }
+  },
+  "qualityStatus": "SUCCESS",
+  "qualitySeverities": [
+    {
+      "severity": {
+        "code": "sanity-check",
+        "position": 1,
+        "name": "Sanity Check",
+        "shortName": "Sanity Ch.",
+        "initials": "S.C.",
+        "defaultOnMissing": false
+      },
+      "scenarioCounts": {
+        "total": 10,
+        "failed": 5,
+        "passed": 5
+      },
+      "percent": 100,
+      "status": "SUCCESS"
+    },
+    {
+      "severity": {
+        "code": "high",
+        "position": 2,
+        "name": "High",
+        "shortName": "High",
+        "initials": "High",
+        "defaultOnMissing": true
+      },
+      "scenarioCounts": {
+        "total": 15,
+        "failed": 2,
+        "passed": 13
+      },
+      "percent": 100,
+      "status": "SUCCESS"
+    },
+    {
+      "severity": {
+        "code": "medium",
+        "position": 3,
+        "name": "Medium",
+        "shortName": "Medium",
+        "initials": "Med.",
+        "defaultOnMissing": false
+      },
+      "scenarioCounts": {
+        "total": 8,
+        "failed": 0,
+        "passed": 8
+      },
+      "percent": 100,
+      "status": "SUCCESS"
+    },
+    {
+      "severity": {
+        "code": "*",
+        "position": 2147483647,
+        "name": "Global",
+        "shortName": "Global",
+        "initials": "Global",
+        "defaultOnMissing": false
+      },
+      "scenarioCounts": {
+        "total": 33,
+        "failed": 7,
+        "passed": 26
+      },
+      "percent": 100,
+      "status": "SUCCESS"
+    }
+  ],
+  "duration": 0,
+  "estimatedDuration": 0,
+  "previousId": 3,
+  "nextId": 5,
+  "countryDeployments": [
+    {
+      "id": 78,
+      "country": {
+        "code": "fr",
+        "name": "France"
+      },
+      "platform": "integ",
+      "jobUrl": "https://build.company.com/demo/deploy/fr/51/",
+      "status": "DONE",
+      "result": "SUCCESS",
+      "startDateTime": "2020-02-27T08:00:00.000Z",
+      "estimatedDuration": 0,
+      "duration": 0
+    }
+  ],
+  "runs": [
+    {
+      "id": 215,
+      "country": {
+        "code": "fr",
+        "name": "France"
+      },
+      "type": {
+        "code": "api",
+        "name": "Integ. APIs",
+        "source": {
+          "code": "api",
+          "name": "API",
+          "letter": "A",
+          "technology": "POSTMAN",
+          "vcsUrl": "https://github.com/decathlon/ara/tree/{{branch}}/server/src/main/resources/demo/collections/",
```

Comment:
please make it more generic, like for example: https://vcs-url.com/company/tree/{{branch}}/collections